src/Parser/AfflictionWarlock/Modules/Talents/AbsoluteCorruption.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import getDamageBonus from '../WarlockCore/getDamageBonus';
+
+const AC_DAMAGE_BONUS = .25;
+
+class AbsoluteCorruption extends Module {
+  bonusDmg = 0;
+
+  on_initialized() {
+    if(!this.owner.error){
+      this.active = this.owner.selectedCombatant.hasTalent(SPELLS.ABSOLUTE_CORRUPTION_TALENT.id);
+    }
+  }
+
+  on_byPlayer_damage(event) {
+    if(event.ability.guid !== SPELLS.CORRUPTION_DEBUFF.id)
+      return;
+    this.bonusDmg += getDamageBonus(event, AC_DAMAGE_BONUS);
+  }
+
+  statistic() {
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.ABSOLUTE_CORRUPTION_TALENT.id} />}
+        value={`${formatNumber(this.bonusDmg)}`}
+        label={'Damage contributed'}
+        tooltip={`Your Absolute Corruption talent contributed ${formatNumber(this.bonusDmg)} total damage (${this.owner.formatItemDamageDone(this.bonusDmg)}).`}
```

Comment:
Showing the same number twice doesn't add anything. I think the best way would be to show either the DPS or the percentage of the total damage contributed (so that it's about the same on short and long fights). I'm not sure which of those two would be the best default display.